app/frontend/components/notes_tagged_verse_export.js

```diff
@@ -0,0 +1,128 @@
+/* This file is part of Ezra Bible App.
+
+   Copyright (C) 2019 - 2021 Ezra Bible App Development Team <contact@ezrabibleapp.net>
+
+   Ezra Bible App is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 2 of the License, or
+   (at your option) any later version.
+
+   Ezra Bible App is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with Ezra Bible App. See the file LICENSE.
+   If not, see <http://www.gnu.org/licenses/>. */
+
+const exportController = require('../controllers/export_controller.js');
+
+/**
+ * The TaggedVerseExport component implements the export of tagged verses into a Word document.
+ * 
+ * @category Component
+ */
+class NotesTaggedVerseExport {
```

Comment:
To make it simply, maybe we should just name it DocExport?

Reply:
done